content/json-ld/latest/events/event-bookingavailability.md

```diff
@@ -0,0 +1,221 @@
+
+When UiTdatabank calculates the bookingAvailability for the top event level, the following logic is being applied:
+
+| bookingAvailability subEvents | Derived bookingAvailability on top event level |
+| ------------- | ---------------- | 
+| All available | available | 
+| All unavailable | unavailable | 
+| Mixed | available | 
+
+
+### HTTP request
+
+```
+PATCH /events/{eventId}/subEvents
+```
+
+Update the bookingAvailability of one or more subEvents.
+
+### Resource properties
+
+| Property	| Type | Description | Example |
+|--|--|--|--|
+| eventId	| uuid | unique identifier for an event | d595414a-13e0-4dd2-b4bd-706598427351 |
+
+### Request body
+
+| Property	| Type | Description | Example |
+|--|--|--|--|
+| id	| integer | a zero-based number of the subEvent you want to set the status for | 2 (this corresponds with the 3th subEvent of your event) |
+| bookingAvailability	| object |  |  |
+| type	| string | bookingAvailability of the event. possible vallues: `Available`, `Unavailable` | Unavailable |
+
+
+### Response
+
+* `204 No Content` : request successful
+* `400 Bad Request` : incorrect method, payload or URI
+* `401 Unauthorized` : expired JWT or the user behind the JWT does not have permission to perform this request
+
+### Example
+
+**request**
+
+Event with calendarType `single` (containing one subEvent)
+
+```
+PATCH https://io-test.uitdatabank.be/events/6ecadd49-ca7b-4204-a8a6-46f786cd3c1e/subEvents
+Content-Type: application/json
+Authorization: Bearer {token}
+X-Api-Key: {apiKey}
+
+[
+   {
+      "id":0,
+      "bookingAvailability":{
+         "type":"Unavailable",
+      }
+   }
+]
+```
+
+For events with calendarType `single`, the derived bookingAvailability for the top event will always be equal to the bookingAvailability of the subEvent. Thus, in the example above the derived bookingAvailability for the top event will become `Unavailable`
+
+Event with calendarType `multiple` (containing 3 subEvents)
+```
+PATCH https://io-test.uitdatabank.be/events/03116768-1abc-405a-93d7-ba6ede52fe09/subEvents
+Content-Type: application/json
+Authorization: Bearer {token}
+X-Api-Key: {apiKey}
+
+[
+  {
+    "id": 0,
+	"bookingAvailability": {
+	  "type": "Unavailable",
+	}
+  },
+  {
+    "id": 1,
+	"bookingAvailability": {
+	  "type": "Unavailable",
+	}
+  },
+    {
+    "id": 2,
+	"bookingAvailability": {
+	  "type": "Available",
+	}
+  }
+]
+```
+Since 2 subEvents have the status `Unavailable` and one subEvent `Available`, the derived status for the top event will become `Available`
+
+**Response**
+
+The following are example responses.
+
+```
+204 No Content
+```
+
+```
+400 Bad Request
+
+{
+    "validation_messages": {
+        "type": "Invalid type provided"
+    },
+    "title": "Invalid payload.",
+    "type": "about:blank"
+}
+```
+
+```
+401 Unauthorized
+
+Token claims validation failed. This most likely means the token is expired.
+```
```

Comment:
400 and 401 examples should probably also be changed here (see previous comment)